types/src/jwks/rsa/mod.rs

```diff
@@ -21,6 +25,45 @@ pub struct RSA_JWK {
     pub n: String,
 }
 
+#[derive(Debug, Deserialize, Serialize)]
+struct RsaJwkSet {
+    keys: Vec<RSA_JWK>,
+}
+
+pub fn get_jwk_from_file(file_path: &str) -> RSA_JWK {
+    let mut file = File::open(file_path).expect(&format!(
+        "Unable to open test jwk json file, path - {}",
+        file_path
+    ));
```

Comment:
does this assume the current working dir is aptos-core root?

i feel `text :&str = include_str!("../path/to/res_file")` does not assume this and is better.

Reply:
cool nice this is better TIL

Reply:
Yeah, we don't want any CWD assumptions... `include_str!` is better. Will you switch to it soon @heliuchuan?

Reply:
done